Trivial (#77467)
ansible/ansible: docs/docsite/rst/community/collection_development_process.rst

Each changelog entry must contain a link to its issue between parentheses at the end. If there is no corresponding issue, the entry must contain a link to the PR itself.

> Most changelog entries are ``bugfixes`` or ``minor_changes``. You can use ``trivial`` as well for any collection that requires a changelog fragment for each pull request. ``trivial`` changelog fragments are excluded from the changelog output.

Reviewer: Most changelog entries are ``bugfixes`` or ``minor_changes``. You can also use ``trivial`` for any collection that requires a changelog fragment for each pull request. The ``trivial`` changelog fragments are excluded from the changelog output.
```suggestion
Most changelog entries are ``bugfixes`` or ``minor_changes``. You can use ``trivial`` as well for any collection that requires a changelog fragment for each pull request. ``trivial`` changelog fragments are excluded from the changelog output.
```

> Most changelog entries are ``bugfixes`` or ``minor_changes``. You can use ``trivial`` as well for any collection that requires a changelog fragment for each pull request. ``trivial`` changelog fragments are excluded from the changelog output.

Reviewer: ```suggestion
Most changelog entries are ``bugfixes`` or ``minor_changes``. You can also use ``trivial``  for any collection that requires a changelog fragment for each pull request. ``trivial`` changelog fragments are excluded from the changelog output.

```
